pkg/generator.go

```diff
@@ -24,6 +24,8 @@ import (
 	"golang.org/x/tools/imports"
 )
 
+const mockConstructorParamTypeNamePrefix = "mockConstructorTestingT"
```

Comment:
Sorry I actually didn't mean we couldn't use the random string idea. I'm just curious if it would make sense to make it `MockConstructorTestingT` instead of `mockConstructorTestingT`. I'm thinking mainly of linters being able to see the interface (I don't actually know if they won't see unexported interfaces).

Reply:
I'm not sure, I've never written a linter for Go before :D But I'm quite sure they don't care about stuff being exported or not - that would make them a shitty linter IMO :rofl: But yeah, making it public later is easy :)